components/playground/instance/ticdc.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instance
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/pingcap/tiup/pkg/repository/v0manifest"
+	"github.com/pingcap/tiup/pkg/utils"
+)
+
+// TiCDC represent a ticdc instance.
+type TiCDC struct {
+	instance
+	pds []*PDInstance
+	Process
+}
+
+var _ Instance = &TiCDC{}
+
+// NewTiCDC create a TiCDC instance.
+func NewTiCDC(binPath string, dir, host, configPath string, id int, pds []*PDInstance) *TiCDC {
+	ticdc := &TiCDC{
+		instance: instance{
+			BinPath:    binPath,
+			ID:         id,
+			Dir:        dir,
+			Host:       host,
+			Port:       utils.MustGetFreePort(host, 8300),
+			ConfigPath: configPath,
+		},
+		pds: pds,
+	}
+	ticdc.StatusPort = ticdc.Port
```

Comment:
where is this `StatusPort` used?

Reply:
this is not used anywhere for now, but I `StatusPort` should be assigned a value